scanpy/preprocessing/_simple.py

```diff
@@ -899,25 +899,37 @@ def subsample(data, fraction=None, n_obs=None, random_state=0, copy=False):
         return X[obs_indices], obs_indices
 
 
-def downsample_counts(adata, target_counts=20000, random_state=0,
-                      replace=True, copy=False):
-    """Downsample counts so that each cell has no more than `target_counts`.
+@deprecated_arg_names({"target_counts": "counts_per_cell"})
+def downsample_counts(
+    adata: AnnData,
+    counts_per_cell: Optional[int] = None,
+    total_counts: Optional[int] = None,
+    random_state: Optional[int] = 0,
+    replace: bool = False,
+    copy: bool = False,
+) -> Optional[AnnData]:
+    """
+    Downsample counts from count matrix.
 
-    Cells with fewer counts than `target_counts` are unaffected by this. This
-    has been implemented by M. D. Luecken.
+    If `counts_per_cell` in specified, each cell will downsampled. If
```

Comment:
Maybe a typo in: If `counts_per_cell` *is* specified?

Reply:
Yep, should definitely be `is`. I've gotta start proofreading my docs better.